src/QsCompiler/CompilerOptimizations/Printer.fs

```diff
@@ -98,7 +98,7 @@ let rec internal printQsTuple (qt: QsTuple<LocalVariableDeclaration<QsLocalSymbo
 
 let rec internal printStm (indent: int) (stm: QsStatementKind): string =
```

Comment:
Is something blocking replacing this file with the standard Q# generation? There seems to be an old branch that I think started to do that but hasn't been updated in a while?

Reply:
The standard Q# generation is not deterministic. The namespace elements are compiled in a random order, so they are printed in a random order, breaking the test cases. I started working on a fix in the Q# code generator, but I've been working on other code recently.

Reply:
Will fix in a future PR